modules/vpc.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: module
+title: Virtual Private Cloud
+image: assets/img/vpc.png
+description: Create and manage VPCs
+---
+Overview
+--------
+Cumulus makes configuring Virtual Private Clouds simpler. Easily configure VPCs, route table definitions, subnets, and endpoint policies. Read the following sections to learn about configuring your VPCs with Cumulus. Example configuration can be found in the [Cumulus repo](https://github.com/lucidsoftware/cumulus).
+
+Virtual Private Cloud
+---------------------
+
+Each VPC is defined in its own file where the file name is the name of the VPC. These files are located in a [configurable](#configuration) folder. VPCs are JSON objects with the following attributes:
+
+* `cidr-block` - the network range for the VPC in CIDR notation e.g. `"10.0.0.0/16"`
+* `tenancy` - the tenancy of instances launched in the VPC. A value of `"default"` means that instances can be launched with any tenancy and a value of `"dedicated"` means instances will launch as dedicated instances regardless of the tenancy assigned to the instance at launch
+* `dhcp` - a JSON object containing DHCP options for the VPC. Updating DHCP options causes all existing and new instances launched in the VPC to use the new options
+  * `domain-name-servers` - an array of up to 4 IP addresses for domain name servers or `"AmazonProvidedDNS"`
+  * `domain-name` - if using `"AmazonProvidedDNS"` in `us-east-1` specifiy `"ec2.internal"`. If using `"AmazonProvidedDNS"` in another region, specify `{region}.compute.internal` (e.g. `"ap-northesat-1.compute.internal"`). Otherwise, specify a domain name
```

Comment:
I can't tell you how much I hate how there's always a special case for `us-east-1`